scylla/src/transport/session_builder.rs

```diff
@@ -982,4 +986,10 @@ mod tests {
         assert!(builder.config.keyspace_case_sensitive);
         assert!(!builder.config.fetch_schema_metadata);
     }
+
+    // This is to assert that #705 does not break the API (i.e. it merely extends it).
+    fn _check_known_nodes_compatibility(hostnames: &[impl AsRef<str>]) {
+        let sb: SessionBuilder = SessionBuilder::new();
+        sb.known_nodes(hostnames);
+    }
```

Comment:
What about a test for `SessionConfig::add_known_nodes_addr` and `SessionConfig::add_known_nodes`? At least the first one has a different signature than what is checked here.